src/responder/pam/pamsrv_p11.c

```diff
@@ -718,10 +719,11 @@ struct tevent_req *pam_check_cert_send(TALLOC_CTX *mem_ctx,
     struct timeval tv;
     int pipefd_to_child[2] = PIPE_INIT;
     int pipefd_from_child[2] = PIPE_INIT;
-    const char *extra_args[16] = { NULL };
+    const char *extra_args[20] = { NULL };
```

Comment:
Why not +2?

Reply:
I thought this was an over-estimation of size, but maybe not. I set it to '18'